tests/gis_tests/geoapp/test_functions.py

```diff
@@ -147,6 +147,29 @@ def test_assvg(self):
         self.assertEqual(svg1, City.objects.annotate(svg=functions.AsSVG('point')).get(name='Pueblo').svg)
         self.assertEqual(svg2, City.objects.annotate(svg=functions.AsSVG('point', relative=5)).get(name='Pueblo').svg)
 
+    @skipUnlessDBFeature('has_AsWKB_function')
+    def test_aswkb(self):
+        wkb = City.objects.annotate(
+            wkb=functions.AsWKB(Point(1, 2, srid=4326)),
+        ).first().wkb
+        # WKB is either XDR or NDR encoded.
+        self.assertIn(
+            bytes(wkb),
+            (
+                b'\x00\x00\x00\x00\x01?\xf0\x00\x00\x00\x00\x00\x00@\x00\x00'
+                b'\x00\x00\x00\x00\x00',
+                b'\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\xf0?\x00\x00'
+                b'\x00\x00\x00\x00\x00@',
+            ),
+        )
+
+    @skipUnlessDBFeature('has_AsWKT_function')
+    def test_aswkt(self):
+        wkt = City.objects.annotate(
+            wkt=functions.AsWKT(Point(1, 2, srid=4326)),
+        ).first().wkt
+        self.assertEqual(wkt, 'POINT (1.0 2.0)' if oracle else 'POINT(1 2)')
```

Comment:
@felixxm 
IMHO this change makes the test unreasonably fragile.

Reply:
Why? IMO using regexp is less readable. This is not a really complicated comparison, I can imagine only one case when this can be broken in the future i.e. when Oracle unify their implementation with other dbs. Honestly I would like to catch this.

Reply:
> I can imagine only one case when this can be broken in the future i.e. when Oracle unify their implementation with other dbs

Or if we add another backend, or if the new version of some of backends will output space after `POINT` (by design or due to some bug). Even in the case you mentioned, we'll modify this code twice: when we add support for the new version and when we remove support for the old version.

> Honestly I would like to catch this.

Why?

Reply:
But actually I'm OK with that.

Reply:
I was close to restore a regexp :wink:, but let's leave it as it is. Thanks :+1: